kytos/core/napps/manager.py

```diff
@@ -79,8 +79,8 @@ def install(self, napp_uri, enable=True):
             return False
 
         if not napp.repository:
-            # Check here what is configured repo
-            pass
+            options = self._controller.options
+            napp.repository = json.loads(options.napps_repositories)[0]
```

Comment:
When I tried to execute the self._controller.options the result is a json, not an array. I will try to change this line.